app/services/streaming/streaming-view.ts

```diff
@@ -156,6 +156,22 @@ export class StreamInfoView<T extends Object> extends ViewHandler<T> {
     return this.streamingState.streamingStatus !== 'offline';
   }
 
+  get isGreen(): boolean {
+    return Services.VideoSettingsService.isGreen;
```

Comment:
this isn't reactive

Reply:
This is intentional for now so that it always returns false and skips over the Green code.